src/main/java/org/openmrs/addonindex/rest/AddOnController.java

```diff
@@ -40,6 +40,11 @@ public AddOnController(Index index) {
 	                                           @RequestParam(value = "q", required = false) String query) throws Exception {
 		return index.search(type, query);
 	}
+
+	@RequestMapping(method = RequestMethod.GET, value = "/api/v1/tags")
```

Comment:
This is the wrong URL for this. It implies a `tag` resource, i.e. you're fetching tags, not the addons. Two options:

1. this would be at "/api/v1/addon?tag=Tag"
2. this would be at "/api/v1/tag/{tagName}/"